PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditRequest.kt

```diff
@@ -0,0 +1,12 @@
+package com.braintreepayments.api.paypal.vaultedit
+
+import com.braintreepayments.api.ExperimentalBetaApi
+
+/**
+ *
+ * Note: **This feature is in beta. It's public API may change in future releases.**
+ */
+@ExperimentalBetaApi
+data class PayPalVaultEditRequest(
+    val encryptedBillingAgreementID: String
```

Comment:
We landed on `edit_paypal_vault_id` per our convos in Slack (#bt-gw-sdk-collaboration)

Reply:
thanks for calling this out! i just updated the property name.